game-core/src/main/java/games/strategy/engine/framework/startup/ui/LocalServerAvailabilityCheck.java

```diff
@@ -12,15 +11,20 @@
  */
 @Builder
 public class LocalServerAvailabilityCheck {
-  private final ILobbyGameController controller;
-  private final INode localNode;
+  private final ConnectivityCheckClient connectivityCheckClient;
+  private final int localPort;
   private final Consumer<String> errorHandler;
 
+  /**
+   * Starts a thread that requests the server to do a 'reverse' connection back to the local client.
+   * This is to ensure that our local IP address is public facing and that other computers on the
+   * internet can establish connections to this (the local) host.
+   */
   public void run() {
     // if we lose our connection, then shutdown
     new Thread(
             () -> {
-              if (!controller.testGame(localNode)) {
```

Comment:
Hmm I'm not seeing that this method is removed, are there any usages?

Reply:
Casualty of working on this from multiple machines, change was made but did not make it into this branch. Now removed: 949f6e9b8bd32f243cd5c89c839186c5f335cb12